crates/libeq_wld/src/parser/fragments/light_source.rs

```diff
@@ -1,8 +1,10 @@
 use std::any::Any;
```

Comment:
This fragment was off. I was able to find it in the old eq alpha game data and fill it out!